tests/components/roborock/test_config_flow.py

```diff
@@ -239,8 +239,11 @@ async def test_reauth_flow(
     assert result["step_id"] == "reauth_confirm"
```

Comment:
Side note: We normally use `MockConfigEntry.start_reauth_flow` to start a reauth flow. Here on line 234 we don't do that.

The return value `result` is also not accurate, as that's always `None` for the used method.

Reply:
I'll take a look at that tomorrow. I think the mocks should also get some attention, because it seems that the library classes are not being properly mocked, but I am not too familiar with the integration yet.